src/nope.ts

```diff
@@ -0,0 +1,61 @@
+import { appendErrors, transformToNestObject, Resolver } from 'react-hook-form';
+import NopeObject from 'nope-validator/lib/umd/NopeObject';
+
+const parseNopeSchema = (error: any, validateAllFieldCriteria: boolean) => {
```

Comment:
Is this possible to avoid any?

```ts
import { ShapeErrors } from 'nope-validator/lib/umd/types';

const parseNopeSchema = (error: ShapeErrors, validateAllFieldCriteria: boolean) => {
```

Reply:
It might be more complicated than casting it to `ShapeErrors`, please see my comment below!